jobspec/test-fixtures/basic.hcl

```diff
@@ -18,6 +18,11 @@ job "binstore-storagelocker" {
   update {
     stagger      = "60s"
```

Comment:
I am actually keeping this till it is fully deprecated because old jobs that we still support will have this key!